moped-editor/src/views/projects/projectView/ProjectNameComplex.js

```diff
@@ -0,0 +1,292 @@
+import React, { useState } from "react";
+import makeStyles from "@mui/styles/makeStyles";
+import { Grid, Icon, Box, Typography, TextField, Snackbar, Alert } from "@mui/material";
+import ProjectStatusBadge from "./ProjectStatusBadge";
+import { useMutation } from "@apollo/client";
+import { UPDATE_PROJECT_NAMES_QUERY } from "../../../queries/project";
+
+const useStyles = makeStyles((theme) => ({
+  editIcons: {
+    cursor: "pointer",
+    marginTop: "16px",
+    fontSize: "2rem",
+  },
+  projectNameEditField: {
+    "font-size": "1.5rem",
+    "font-weight": "bold",
+  },
+  projectSecondaryNameEditField: {
+    "font-size": "1.0rem",
+    "font-weight": "bold",
+    "padding-top": "0.95rem",
+  },
+}));
+
+const ProjectNameComplex = (props) => {
```

Comment:
I'm very open to suggestions on a better name here. I think of a complex as a set of related things, but maybe that's just for buildings. I'm still thinking about it -- anything y'all would prefer?

Reply:
Hey Mike,

Thanks again for all your guidance on this Component. I have pulled out the [commonJSX, editModeJSX and viewModeJSX into proper components](https://github.com/cityofaustin/atd-moped/blob/65b04759ea0db073c9942f3f06f584d8c57257e2/moped-editor/src/views/projects/projectView/ProjectNameComplex.js#L168-L320). Do you think I should put them in their own files or leave them in the same file as the main ProjectNameComplex component. I don't think we have an eye to reuse them, so I thought I would ask what you think is best and follow that direction. Thanks!!

Reply:
@frankhereford sorry, i missed this in my notifications! We don't have a super strong pattern here, but I was looking at the project phase components called `ProjectPhases`, `ProjectPhaseForm`, etc and feel like something like `ProjectNameEditable`, `ProjectNameEditableForm`, and `ProjectName` could work if these are split into separate files.

Also, for the common JSX, that snackbar feels like a pretty good fit to stay put and not be its own component to me since it is small amount of code. Total judgement call, though!

Reply:
Mike, these are great suggestions!! Thank you so much for all of your help. I've adopted each of these. 🚀 https://github.com/cityofaustin/atd-moped/pull/1302/commits/180f5502c46bd0205e96b84576de7675995497ca